core/src/com/mygdx/game/steerers/FollowPathSteerer.java

```diff
@@ -36,6 +43,45 @@
  */
 public class FollowPathSteerer implements Steerer {
 
+	// This should become part of gdx-ai since the ability to know
+	// the selected behavior is generally useful.
+	private static class MyPrioritySteering extends PrioritySteering<Vector3> {
+		protected int selectedBehaviorIndex;
+
+		public MyPrioritySteering (Steerable<Vector3> owner, float epsilon) {
+			super(owner, epsilon);
+		}
+		
+		public int getSelectedBehaviorIndex() {
```

Comment:
I'm going to put this in gdx-ai.
It returns the index of the matched behavior.
Do you think method name is appropriate?